tools/clang/test/SemaHLSL/rayquery-decls-errors-TU-export.hlsl

```diff
@@ -0,0 +1,6 @@
+// RUN: %dxc -T lib_6_5 -verify %s
+
+export float4 MyExportedFunction(float4 color) {
```

Comment:
Should have the globals in this file as well to verify that they are not picked up by the export function for diagnostics.